pig/src/main/java/org/jboss/pnc/bacon/pig/impl/addons/quarkus/QuarkusCommunityDepAnalyzer.java

```diff
@@ -271,7 +273,7 @@ private Path generateQuarkusProject(Predicate<String> artifactIdSelector, String
     private String devtoolsJarName() {
         String bomArtifactId = getBomArtifactId();
         if (!isProductBom(bomArtifactId)) {
-            return "quarkus-bom-descriptor-json-" + quarkusVersion + ".json";
+            return "quarkus-bom-quarkus-platform-descriptor-" + quarkusVersion + "-" + quarkusVersion + ".json";
```

Comment:
I think now the name of `bom-desciptor-json` will depend on artifact now 
it will be of form 
<bom-artifact-id>-quarkus-platform-descriptor-<quarkusVersion>-<quarkusVersion>.json
we need to include <bom-artifact-id> now in this

Reply:
But this is just to find the json file and not else. Would this still affect us?